index.js

```diff
@@ -53,17 +53,19 @@ const setupMix = async () => {
   let content = fs.readFileSync(packagePath);
```

Comment:
If `--no-scripts` is specified, it is a waste to read the file and write it back doing nothing to it, so in that case it would be better to move these within the `if(program.scripts)`. However, in the future there might be other things we'd like to write to the package.json. Should I move these inside the if now, and then move it out in the future should the case arrive, or just leave it like this? What do you think, @ranbogmord

Reply:
Since the `--output` flag was added we now have the `pkgjson` variable that could be used instead of `content` and `pack`. That would mean that we only read the pkgjson once, what do you think?

Reply:
So this would be something like ```
pkgjson.scripts = Object.assign((pkgjson.scripts || {}), {
    ...
})
```

Reply:
`pkgjson` is our package though, we don't want to add the build script too it :)

Reply:
Oh I'm so dumb, ofc... Yeah, we could move the loading into `if (program.scripts)` then, shouldn't be needed outside of it atm.

Reply:
Done :)